### PR TITLE
Update CI to publish wheels

### PR DESCRIPTION
**Pull request by xadupre**
Still strugling with the manylinux build with Centos...

### CODE REVIEW

**Comment by TomWildenhain-Microsoft** [comment]
Yay!  Is it on test pypi?

**Comment by xadupre** [comment]
> Yay! Is it on test pypi?

I'll do that today after I test it on windows.

**Comment by xadupre** [comment]
> > Yay! Is it on test pypi?
> 
> I'll do that today after I test it on windows.

Published at https://test.pypi.org/project/onnxruntime-customops/0.0.1/. 

I'm still unable to build the manylinux wheel. The docker provided by cibuildwheel does not have cmake>=3.16 so it needs to be built but this cmake does not find python libraries. I suggest to merge this PR and do another one to fix it. The wheels on windows and macosx are built at every commit. The build for linux is also build but it cannot be published on pypi as pypi only accepts builds for many linux.

File: ci_build/azure-pipelines/mshost.yaml [hunk 1]
```diff
@@ -1,229 +1,339 @@
...
+      destinationFolder: '$(Build.sourcesdirectory)'
+      cleanDestinationFolder: false
+      overwriteExistingFiles: true
+    displayName: Unpack ONNXRuntime package.
+
+  - script: |
+      python -m pip install cibuildwheel
+      export CIBW_BEFORE_BUILD="pip install pybind11 cython numpy scipy scikit-learn onnx onnxruntime cmake"
+      # Binaries of cmake remains stuck.
+      # export CIBW_BEFORE_BUILD_LINUX="yum update && yum install -y wget && mkdir _install_ && cd _install_ && wget https://github.com/Kitware/CMake/releases/download/v3.20.1/cmake-3.20.1-linux-x86_64.sh && bash  cmake-3.20.1-linux-x86_64.sh && export PATH=./_install_/bin:$PATH"
```
Comment (wenbingl) [inline ci_build/azure-pipelines/mshost.yaml:42]: this is an ubuntu image, should it be apt, instead of yum?

**Comment by xadupre** [comment]
I'll remove it. I'll open an another PR for it.

File: ci_build/azure-pipelines/mshost.yaml [hunk 2]
```diff
@@ -1,229 +1,327 @@
...
+      py37-170:
+        python.version: '3.7'
+        ort.version: '1.7.0'
+      py36-170:
+        python.version: '3.6'
+        ort.version: '1.7.0'
+    maxParallel: 1
+
+  steps:
+  - script: conda create --yes --quiet --name pyenv -c conda-forge python=3.8 numpy
```
Comment (wenbingl) [inline ci_build/azure-pipelines/mshost.yaml:289]: python=3.8?

File: MANIFEST.in [hunk 3]
```diff
@@ -0,0 +1,5 @@
+prune ci_build
+prune docs
+exclude *.bat
+exclude *.yml
```
Comment (wenbingl) [inline MANIFEST.in:4]: yaml?

File: CMakeLists.txt [hunk 4]
```diff
@@ -151,7 +151,9 @@ if(OCOS_ENABLE_PYTHON)
   if(NOT "${Python3_FIND_REGISTRY}" STREQUAL "NEVER")
     message(FATAL_ERROR "Python3_FIND_REGISTRY is not NEVER")
   endif()
+  message("-- Find Python3")
```
Comment (Toudsour) [inline CMakeLists.txt:154]: Maybe we could delete one of message("-- Find Python3")

Comment (xadupre) [reply CMakeLists.txt]: done

**Review by wenbingl — APPROVED** [review]

**Review by TomWildenhain-Microsoft — APPROVED** [review]

**Pull request merged**